Source/Blazorise/Models/RatingItemTooltipInfo.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace Blazorise;
+
+/// <summary>
+/// Defines the tooltip information for the <see cref="Rating"/> component.
+/// </summary>
+public class RatingItemTooltipInfo
```

Comment:
What about **RatingItemTooltip** You are defining how the actual tooltip gets constructed not just the "info".

Reply:
Renamed it to `RatingTooltip`. We already don't actively expose `RatingItem`, only the `Rating` component. So this makes the most sense.